Content.Shared/Armable/ArmableSystem.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Shared.Examine;
+using Content.Shared.Toggleable;
+using Content.Shared.Item.ItemToggle.Components;
+
+namespace Content.Shared.Armable;
+
+public sealed class ArmableSystem : EntitySystem
+{
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ArmableComponent, ExaminedEvent>(OnExamine);
+        SubscribeLocalEvent<ArmableComponent, ItemToggleActivateAttemptEvent>(TryArming);
+        SubscribeLocalEvent<ArmableComponent, ItemToggledEvent>(ArmingDone);
+    }
+
+    private void TryArming(Entity<ArmableComponent> entity, ref ItemToggleActivateAttemptEvent args)
```

Comment:
I was under the impression that `ItemToggleActivateAttemptEvent` is meant for canceling the event if a condition is not made (e.g. the landmine is already armed, so the event should be canceled), not to actually toggle any functionality.

You would put the toggle functionality in the `ItemToggledEvent` instead. 

This would also let you skip the weird `comp.OnActivate = false;` thing that you have in `ArmingDone`, which doesn't have similar functionality anywhere else in the code-base.

Reply:
I have moved the statements to their correct method but the purpose of `comp.OnActivate = false;` is to remove the deactivate verb. I am not sure how to otherwise do that.

Reply:
Ah, then it should be fine to keep it there.

Reply:
Is it actually possible to try to activate it twice? Should I keep a check if it is already activated in the `TryArming`-method? If not, then the method would be unnecessary and could be cut.